commands.sh

```diff
@@ -7,7 +7,7 @@ cd PhysObjectExtractorTool
 cd PhysObjectExtractor
 scram b
 
-sed -i "s/process.GlobalTag.connect/#process.GlobalTag.connect/g" python/poet_cfg.py
+# sed -i "s/process.GlobalTag.connect/#process.GlobalTag.connect/g" python/poet_cfg.py
```

Comment:
Currently this line does not do anything because it is commented out in the config file.  But, I think it is better to leave it uncommented, just in case.